index.html

```diff
@@ -5,10 +5,12 @@
     <meta charset="utf-8">
     <title>Twisted</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/x-icon" href="assets/images/favicon.ico">
     <link href="assets/css/bootstrap-v4.5.3.min.css" rel="stylesheet" type="text/css" media="all" />
     <link href="assets/css/bootstrap-icons.css" rel="stylesheet" type="text/css" media="all" />
     <link href="assets/css/default.min.css" rel="stylesheet" type="text/css" media="all" />
     <link href="assets/css/style.css" rel="stylesheet" type="text/css" media="all" />
+    <link href="assets/css/tailwind.css" rel="stylesheet" type="text/css" media="all" />
```

Comment:
do we need tailwind.css for the index page?

Reply:
no